darwin/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Xcode
+xcuserdata
+DerivedData/
+build/
+# build.sh
+XCSoarExecutable
```

Comment:
Newiline

Reply:
Will address in https://github.com/XCSoar/XCSoar/pull/1713.